code/src/components/TodoItem.js

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import styled from "styled-components";
+import { useDispatch } from "react-redux";
+import { todos } from "reducers/todos";
+import moment from "moment";
+
+const ItemContainer = styled.div`
+  padding: 8px;
+  margin: 25px;
+  margin-bottom: 2px;
+  font-size: 24px;
+  color: #ffff;
+  background-color: #266150;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  border-radius: 5px;
+  cursor: pointer;
+`;
+
+const ItemText = styled.div``;
+
+const TodoComplete = styled.input`
+  :checked {
+    text-decoration: line-through;
+  }
```

Comment:
I think this line of code is not being executed. You're using the `text-decoration: line-though` property with the in-line styling in line 58 below: `style={{ textDecoration: item.isCompleted ? "line-through" : "" }}`